doc/subsystems/jinjagen/jinjagen.rst

```diff
@@ -0,0 +1,223 @@
+..
+    Copyright (c) 2004-2015 Ned Batchelder
+    SPDX-License-Identifier: MIT
+    Copyright (c) 2018 Bobby Noelte
+    SPDX-License-Identifier: Apache-2.0
+    Copyright (c) 2018 Nordic Semiconductor ASA
+    SPDX-License-Identifier: Apache-2.0
+.. _codegen_intro:
+
+Introduction
+============
```

Comment:
Our project convention is for headings to start with a ``###`` underline (for the title), followed by ``***``, ``===``, and ``---`` for nested levels.

Reply:
I was looking for some documentation for this, but ended up just following the Sphinx documentation. I'll update the topics

Reply:
Ok